documentation/intermediate_ca.md

```diff
@@ -55,7 +55,7 @@ Puppet Server needs to present the full certificate chain to clients so the clie
 -----END X509 CRL-----
 ```
 
-8. Use the `puppet server can import` command to trigger the rest of the CA setup:
+8. Use the `puppet server import` command to trigger the rest of the CA setup:
```

Comment:
sorry, I think that if this is in backticks it usually denotes the command, which is `puppetserver ca import` but if you want to refer to "the puppet server import command to.." it doesn't need to be wrapped in backticks.

Reply:
Updated, thanks!